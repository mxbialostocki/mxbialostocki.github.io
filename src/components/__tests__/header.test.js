import React from "react"
import renderer from "react-test-renderer"

import Header from '../header'

describe("layout", () => {
  it("Renders correctly", () => {
    const tree = renderer.create(<Header siteTitle="mxbialostocki" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})