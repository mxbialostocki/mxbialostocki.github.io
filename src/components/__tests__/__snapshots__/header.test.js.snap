// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout Renders correctly 1`] = `
<header
  className="makeStyles-header-3"
>
  <div
    className="makeStyles-headerWrap-5"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3 MuiGrid-grid-xl-3"
      >
        <a
          href="/"
        >
          <h5
            className="MuiTypography-root MuiTypography-h5"
          >
            mxbialostocki
          </h5>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 MuiGrid-grid-md-3 MuiGrid-grid-xl-3"
      >
        <a
          href="/photography"
        >
          <h5
            className="MuiTypography-root MuiTypography-h5"
          >
            photography
          </h5>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 MuiGrid-grid-md-3 MuiGrid-grid-xl-2"
      >
        <a
          href="/code"
        >
          <h5
            className="MuiTypography-root MuiTypography-h5"
          >
            code
          </h5>
        </a>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-md-3 MuiGrid-grid-xl-4"
      >
        <a
          href="/contact"
        >
          <h5
            className="MuiTypography-root makeStyles-lastChild-4 MuiTypography-h5"
          >
            contact
          </h5>
        </a>
      </div>
    </div>
    <div
      className="makeStyles-headerContainer-9"
    />
  </div>
</header>
`;
